order-service/order-service.ts

```diff
@@ -0,0 +1,201 @@
+import { Tx, Pair, Position, OrderConfig } from '../types/common.js';
+import {
+  OrderSide,
+  OrderType,
+  BECH32_PREFIX,
+  LocalWallet,
+  SubaccountClient,
+  Network,
+  CompositeClient,
+  IndexerClient,
+} from '@dydxprotocol/v4-client-js';
+
+// order factory function
+const createOrderService = async (mnemonic: string, network: Network) => {
+  const wallet = await LocalWallet.fromMnemonic(mnemonic, BECH32_PREFIX);
+  const subaccount = new SubaccountClient(wallet, 0);
+  const client = await CompositeClient.connect(network);
+
+  const indexer_client = new IndexerClient(network.indexerConfig);
+
+  return {
+    placeLimitOrder: (
+      pair: Pair,
+      side: OrderSide,
+      price: number,
+      size: number,
+      config: OrderConfig,
+    ) => {
+      return new Promise<Tx>((resolve) => {
+        resolve(
+          client.placeOrder(
+            subaccount,
+            pair,
+            OrderType.LIMIT,
+            side,
+            price,
+            size,
+            config.clientId,
```

Comment:
What's this? How the user should pass this value?

Reply:
is a number, can be used by the client to identify the order

Reply:
Then we should manage it manually. We can talk about it in our meeting.